### PR TITLE
Bot reminder 2.0

### PR DESCRIPTION
**Pull request by Evgen2325**

### CODE REVIEW

File: bot.py [hunk 1]
```diff
@@ -21,24 +22,27 @@ def get_translate_func(message):
...
-    # TODO add assertion for string
-    user_date = message.text.split("-")[0]
-    date_description = message.text.split("-")[1]
-    db.set_user_date(message.chat.id, user_date, date_description)
-    # bot.send_message(message.chat.id, f"wrong input, you need input like (example '22/12/2023-mothers day')\n")
-
+def add_user_dates_in_db(message):
+    check_dates = message.text
+    if re.match(r'\d{2}.\d{2}.\d{4}-.*$', check_dates):
+        user_date = message.text.split("-")[0]
```
Comment (samodur69) [inline bot.py:27]: you already has variable check_dates. You can split it

File: bot.py [hunk 2]
```diff
@@ -21,24 +22,27 @@ def get_translate_func(message):
...
+        date_description = message.text.split("-")[1]
+        db.set_user_date(message.chat.id, user_date, date_description)
+        bot.send_message(message.chat.id, f'Your date was add')
```
Comment (samodur69) [inline bot.py]: your date was add**ED**

File: bot.py [hunk 3]
```diff
@@ -59,25 +63,38 @@ def send_translated_message(message):
...
-    result_string = ''
-    dates_for_user = db.get_from(message.chat.id)
-    for single_date in dates_for_user:
-        result_string += single_date + "\n"
-    bot.send_message(message.chat.id, result_string)
+def look_at_all_user_dates(message):
+    dates = []
+    dates_of_user = db.get_from(message.chat.id)
+    for single_date in dates_of_user:
+        date_and_description = str(single_date[0]) + ")" + ' ' + single_date[2] + ' ' + single_date[3]
```
Comment (samodur69) [inline bot.py]: why you use single_date[0]? its id from DB. List of dates is enough

**Comment by Evgen2325** [comment]
3 weeks work ready)

**Pull request merged**